Internal Server Error: /users/
Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedColumn: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/defaulttags.py", line 320, in render
    if match:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 412, in __bool__
    self._fetch_all()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^

"GET /users/ HTTP/1.1" 500 210107
Internal Server Error: /users/
Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedColumn: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/defaulttags.py", line 320, in render
    if match:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 412, in __bool__
    self._fetch_all()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^

"GET /users/ HTTP/1.1" 500 210748
Internal Server Error: /users/
Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedColumn: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/defaulttags.py", line 320, in render
    if match:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 412, in __bool__
    self._fetch_all()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^

Internal Server Error: /users/
Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedColumn: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/defaulttags.py", line 320, in render
    if match:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 412, in __bool__
    self._fetch_all()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^

Internal Server Error: /users/create/
Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedColumn: column "last_login" of relation "users_user" does not exist
LINE 1: INSERT INTO "users_user" ("password", "last_login", "is_supe...
                                              ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/views/generic/edit.py", line 184, in post
    return super().post(request, *args, **kwargs)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/views/generic/edit.py", line 153, in post
    return self.form_valid(form)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/contrib/messages/views.py", line 12, in form_valid
    response = super().form_valid(form)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/views/generic/edit.py", line 135, in form_valid
    self.object = form.save()
  File "/home/shalin/Загрузки/python-project-52-main(5)/python-project-52-main/task_manager/users/forms.py", line 44, in save
    user.save()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/contrib/auth/base_user.py", line 76, in save
    super().save(*args, **kwargs)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column "last_login" of relation "users_user" does not exist
LINE 1: INSERT INTO "users_user" ("password", "last_login", "is_supe...
                                              ^

"POST /users/create/ HTTP/1.1" 500 181965
Internal Server Error: /users/
Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedColumn: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/template/defaulttags.py", line 320, in render
    if match:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 412, in __bool__
    self._fetch_all()
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/shalin/.cache/pypoetry/virtualenvs/hexlet-code-AUZoa3BN-py3.10/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column users_user.last_login does not exist
LINE 1: ...ELECT "users_user"."id", "users_user"."password", "users_use...
                                                             ^

"GET /users/ HTTP/1.1" 500 210742
